elk/utils_evaluation/ccs.py

```diff
@@ -1,6 +1,8 @@
+from typing import Literal
 import numpy as np
 import torch
 import matplotlib.pyplot as plt
+from enum import Enum
```

Comment:
Is it used somewhere?

Reply:
I don't think so, it's a vestige from earlier when Fabien was using an `Enum`